public/app/features/expressions/types.ts

```diff
@@ -140,6 +140,10 @@ export interface ClassicCondition {
     params: number[];
     type: EvalFunction;
   };
+  unloadEvaluator?: {
```

Comment:
Hysteresis expression is an extension of Threshold expression, with two evaluators (we add the unloadEvaluator)